src/promptflow/promptflow/_sdk/operations/_run_operations.py

```diff
@@ -228,6 +228,31 @@ def update(
         ORMRun.get(name).update(display_name=display_name, description=description, tags=tags, **kwargs)
         return self.get(name)
 
+    @monitor_operation(activity_name="pf.runs.delete", activity_type=ActivityType.PUBLICAPI)
```

Comment:
@D-W-  shall new activity using pf.run instead of pf.runs. How EPR whiltelist our activity names?